### ISSUE TITLE
Broaden list of state contributors

### ISSUE THREAD

**Issue by emily878** [GSA/datagov-wptheme#400]
I'm excited to see how 18F's projects may drive additional use of Data.gov datasets. However, it reminded me that your distribution of state data sites is quite uneven. Your state (and municipal) contributing organizations appear to skew to data available on Socrata-powered sites. I know Socrata makes it easy to federate their data, but in the interest of completion could you be aim to get data from all states? 

The easiest way to broaden the range of municipal contributors you have might be to offer add "suggest a federal, state or local government data site" to your list of things that people can do through email on your contact page. 

Thanks for considering!

**Comment by JeanneHolm** [comment]
Emily--completely agree and the skew you mention is an artifact of our transition to the new JSON harvesting method, which broke the connection to cities previously federated and running on CKAN, Junar, and other methods.  We'll work on getting those cities back in and continue to reach out to others as well.

**Comment by JeanneHolm** [comment]
@ianjkalin and @chrismetcalf can you provide the links to the new JSON files for the additional Socrata-powered cities.  We will also integrate Open Colorado, Open Philadelphia, Palo Alto, and others.

**Comment by ianjkalin** [comment]
@JeanneHolm Of course!  Socrata is still finalizing some changes to our data.json API and will publish the links as soon as those changes are through the QA process.

**Comment by rebeccawilliams** [comment]
We are looking forward to these updates @ianjkalin. 

Also, for what it is worth [State GIS information](http://gisinventory.net/about-the-gis-inventory) is federated on to Data.gov via: http://catalog.data.gov/harvest/gis-inventory-ramona-nsgic.

**Comment by ianjkalin** [comment]
Well, @rebeccawilliams, we took two steps forward and one step back.

First, the data.json API is working from a metadata perspective.  See examples here:

http://labs.data.gov/dashboard/validate?schema=non-federal&output=browser&datajson_url=https%3A%2F%2Fdata.austintexas.gov%2Fdata.json

&

http://labs.data.gov/dashboard/validate?schema=non-federal&output=browser&datajson_url=https%3A%2F%2Fdata.sfgov.org%2Fdata.json

But there appears to be a new problem with duplicate entries from old cross-federation between other portals.  We are resolving that bug now and should have another update soon.

**Comment by rebeccawilliams** [comment]
Ah yes @ianjkalin. Looks like that Issue (#476) has been resolved from the Data.gov end. 

& Love that valid JSON!
